content/cpp/concepts/queues/terms/pop/pop.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.pop()'
+Description: 'Removes the element at the front of the queue.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Objects'
+  - 'OOP'
+  - 'Classes'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `.pop()` method removes the element at the front of the queue. This will be the oldest element in the queue, the element which was added the earliest.
```

Comment:
```suggestion
The `.pop()` method removes the element at the front of the queue. This will be the oldest element in the queue or the element which was added the earliest.
```